ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -1232,7 +1232,7 @@ periodics:
   interval: 48h
   labels:
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: release-openshift-origin-installer-e2e-aws-ovn-4.4
+  name: canary-release-openshift-origin-installer-e2e-aws-ovn-4.4
```

Comment:
Please do not do this.  Just drop the information annotation.

Reply:
Resolved via https://github.com/openshift/release/pull/7768#issuecomment-608642052